lldb/test/API/tools/lldb-dap/progress/Progress_emitter.py

```diff
@@ -37,7 +37,10 @@ def create_options(cls):
         )
 
         parser.add_option(
-            "--total", dest="total", help="Total to count up.", type="int"
+            "--total",
+            dest="total",
+            help="Total to count up, use -1 to identify as indeterminate",
```

Comment:
Don't specify to use -1 as indeterminate, if not set, this will default to `None`, see comment below. You can say "If this option isn't specified with a valid value of 1 or higher, the progress will be indeterminate."